code/src/App.js

```diff
@@ -1,9 +1,42 @@
 import React from 'react'
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom'
+
+import { Nav } from './components/Nav'
+import { Footer } from './components/Footer'
+import { NotFound } from './pages/NotFound'
+import { MovieList } from './components/MovieList'
+import { MoviesTop } from './components/MoviesTop'
+import { MoviesSoon } from './components/MoviesSoon'
+import { MovieDetail } from './pages/MovieDetail'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Nav />
+      <main>
+        <Switch>
+          {/* default: now playing */}
+          <Route path="/" exact>
+            <MovieList />
+          </Route>
+          {/* top rated */}
+          <Route path="/top">
+            <MoviesTop />
+          </Route>
+          {/* coming soon */}
+          <Route path="/soon">
+            <MoviesSoon />
+          </Route>
+          <Route path="/movies/:id" exact>
```

Comment:
No need for `exact` over here, in your setup only `"/"` needs it

Reply:
cleaned this up.